routers/web/repo/setting_protected_branch.go

```diff
@@ -284,3 +285,40 @@ func SettingsProtectedBranchPost(ctx *context.Context) {
 		ctx.Redirect(fmt.Sprintf("%s/settings/branches", ctx.Repo.RepoLink))
 	}
 }
+
+// RenameBranchPost responses for rename a branch
+func RenameBranchPost(ctx *context.Context) {
+	form := web.GetForm(ctx).(*forms.RenameBranchForm)
```

Comment:
I would like a `, ok` check here to ensure that the returned form isn't accidentally of another type.
Because in that case, gitea would most likely panic here which I doubt is wanted.

Reply:
Don't worry, `middleware.Validate` will check form before this handle.